packages/components/psammead-program-card/src/index.jsx

```diff
@@ -0,0 +1,169 @@
+import {
+  C_EBON,
+  C_METAL,
+  C_SHADOW,
+  C_WHITE,
+  C_POSTBOX,
+  C_KINGFISHER,
+} from '@bbc/psammead-styles/colours';
+import {
+  getSansRegular,
+  getSerifRegular,
+  getSansBold,
+} from '@bbc/psammead-styles/font-styles';
+import {
+  getLongPrimer,
+  getMinion,
+  getPica,
+} from '@bbc/gel-foundations/typography';
+import { Link } from '@bbc/psammead-story-promo';
+
+const programStateColors = {
+  live: {
+    backgroundColor: C_POSTBOX,
+    headerLabelColor: C_POSTBOX,
+    headerTextColor: C_EBON,
+    durationColor: C_WHITE,
+  },
+  next: {
+    backgroundColor: C_WHITE,
+    headerLabelColor: C_KINGFISHER,
+    headerTextColor: C_METAL,
+    durationColor: C_KINGFISHER,
+  },
+  onDemand: {
+    backgroundColor: C_EBON,
+    headerLabelColor: '',
+    headerTextColor: C_EBON,
+    durationColor: C_WHITE,
+  },
+};
+
+const CardWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+  background-color: ${C_WHITE};
+  position: relative;
+`;
+
+const TextWrapper = styled.div`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const HeadingWrapper = styled.h3`
+  ${({ service }) => service && getSerifRegular(service)};
+  ${({ script }) => script && getPica(script)};
+  color: ${({ headerTextColor }) => headerTextColor};
+  margin: 0; /* Reset */
+`;
+
+const LabelWrapper = styled.span`
+  ${({ service }) => service && getSansBold(service)};
+  ${({ script }) => script && getPica(script)};
+  text-transform: uppercase;
+  color: ${({ headerLabelColor }) => headerLabelColor};
+`;
+
+const DateWrapper = styled.div`
+  color: ${C_SHADOW};
+  padding: ${GEL_SPACING} 0;
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getPica(script)};
+`;
+
+const SummaryWrapper = styled(Paragraph)`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getLongPrimer(script)};
+  color: ${C_METAL};
+  padding-bottom: ${GEL_SPACING_DBL};
+`;
+
+const ButtonWrapper = styled.div`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getMinion(script)};
+  padding: ${GEL_SPACING} 0;
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  color: ${({ durationColor }) => durationColor};
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${({ durationColor }) => durationColor};
+    fill: currentColor;
+    width: 1.0625rem;
+    height: ${GEL_SPACING_DBL};
+    margin: 0;
+  }
+`;
+
+const DurationWrapper = styled.span`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const ProgramCard = ({
+  dir,
+  service,
+  script,
+  heading,
+  summary,
+  date,
+  duration,
+  state,
+  ctaLink,
+}) => (
+  <CardWrapper>
+    <Link href={ctaLink}>
```

Comment:
According to the accessibility requirements here: https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5dc0724850731722a4aa767a, we should just be wrapping the heading in a link, and using the faux link block pattern that this `Link` component (and the bulletin component) uses to make the whole card clickable.